src/screens/SendTokens.js

```diff
@@ -203,33 +215,73 @@ class SendTokens extends React.Component {
     this.setState({ errorMessage: message, ledgerStep: 0 });
   }
 
+  /**
+   * Method executed before sending a tx to be signed by Ledger
+   *
+   * Checks if we have any custom token, if not we just send to sign
+   * If we have custom tokens we must send all tokens first
+   * When all tokens are sent we should send the tx to sign
+   */
+  beforeSendLedger = () => {
+    // remove HTR if present
+    const txTokens = this.state.txTokens.filter(t => !hathorLib.tokens.isHathorToken(t.uid));
+
+    if (txTokens.length === 0) {
+      // no custom tokens, just send
+      this.executeSendLedger();
+      return;
+    }
+    // get all custom tokens, and include the signatures
+    const tokenSignatures = tokens.getTokenSignatures();
+    const missingSigs = txTokens.filter(t => !tokenSignatures.hasOwnProperty(t.uid));
+    if (missingSigs.length !== 0) {
+      // there are tokens without signatures, missingSigs
+      // set tittle and content
+      this.ledgerAlertModalTitle = t`Unverified custom tokens`
+      this.ledgerAlertModalBody = this.renderAlertTokenList(missingSigs);
+      $("#ledgerAlertModal").modal('show');
+      return;
```

Comment:
If the user is trying to send a tx and a custom token does not have a signature, I thought we would give the possibility for the user to sign it while sending. Isn't it the design?